### PR TITLE
part1

### PR DESCRIPTION
**Pull request by CyborgChameleon**

### CODE REVIEW

File: src/search.js [hunk 1]
```diff
@@ -3,3 +3,37 @@ import React from "react";
 import axios from "axios";
 
 import "./css/song.css";
+export default class SongSearch extends React.Component{
+	constructor(){
+		super()
```
Comment (mreigen) [inline src/search.js]: Let's work on this PR together in class to fix the format (indentation, spaces, etc) before we merge this to the `main` branch

**Comment by lylelung** [comment]
don't forget

import React from "react";

**Comment by mreigen** [comment]
> don't forget
> 
> import React from "react";

@lyleislagging Hey actually I saw that she already included it https://github.com/basic2mastery/shazam-client/blob/f8a1fffe5702d5d7b2ac677ac4050db434054434/src/search.js

File: src/search.js [hunk 2]
```diff
@@ -3,3 +3,37 @@ import React from "react";
 import axios from "axios";
```
Comment (lylelung) [inline src/search.js]: import React from "react";

**Pull request merged**